.github/workflows/ci-checks.yml

```diff
@@ -0,0 +1,12 @@
+name: 'CI checks'
+runs:
+  using: 'composite'
```

Comment:
I had to add svelte-check to our CI jobs here and I've thought that it would be nice to experiment with those composite actions - we'll see how it goes, it's the first time when I've attempted to use this.